batchlib/workflows/cell_analysis.py

```diff
@@ -335,10 +322,8 @@ def core_workflow_tasks(config, name, feature_identifier):
 
     # we could also just add these on demand depending on what we need according to the background params
     bg_estimation_keys = [marker_ana_in_key] + serum_ana_in_keys
-    job_list, bg_tables1 = add_background_estimation_from_min_well(job_list, config,
-                                                                   bg_estimation_keys, feature_identifier)
-    job_list, bg_tables2 = add_background_estimation(job_list, config, config.seg_key,
-                                                     bg_estimation_keys, feature_identifier)
+    job_list = add_background_estimation_from_min_well(job_list, config, bg_estimation_keys)
+    job_list = add_background_estimation(job_list, config.seg_key, bg_estimation_keys)
```

Comment:
This is just personal preference, but all the `add_*` methods act on the `job_list` in place, right? Then I would maybe remove the `job_list = ` to make this more obvious.

Reply:
Yes, they do work in place; but I usually prefer to give it as explicit return value too.
(The implicit reference / not reference arguments is one of my pet peeves with python, so I mostly try to be explicit with return value in cases where it doesn't make the code more unreadable.)